test/test_backend/test_backend.py

```diff
@@ -120,23 +102,54 @@ def test_route_csv_upload_new_cset_version_with_concepts(self):
         self.assertEqual(response.json()['result'], 'success')
 
     # TODO: test is failing. fix
-    @unittest.skip("Skipping test temporarily: It is long-running, and may be moved to a separate GH action")
+    # @unittest.skip("Skipping test temporarily: It is long-running, and may be moved to a separate GH action")
     def test_counts_compare_schemas(self):
```

Comment:
## `test_counts_compare_schemas()` fixes
TODO: updates in progress.

## `counts_compare_schemas()` updates
These handle predictable edge case issues.
- Now returns `None` if is asked to compare against most recent backup but there is no backup.
- If asked to compare a schema that doesn't exist, now throws an error instead of populating 0's in the rows for that schema name